packages/cojson/src/coValueCore/coValueCore.ts

```diff
@@ -60,7 +60,15 @@ export function idforHeader(
   return `co_z${hash.slice("shortHash_z".length)}`;
 }
 
+let logPermissionErrors = true;
```

Comment:
I wouldn't log these by default.

It's very common to have invalid transactions, e.g. child is now an invalid transaction, so it would be very loud

I think it's enough to have these visible in the inspector.

Reply:
That's was the previous behaviour.
Anyway, changing it might confuse people with debugging. Inspector is an advanced tool, but everyone has the console open, and logs can help to understand what's going on.

What if we turn off noisy internal errors, like the child transaction mentioned?